src/v2/email/EmailService.ts

```diff
@@ -0,0 +1,22 @@
+import { MailerService } from '@nestjs-modules/mailer';
+import { Injectable } from '@nestjs/common';
+
+import { EmailOptions } from './dto/EmailDTO';
+
+@Injectable()
+export class EmailService {
+  constructor(private mailerService: MailerService) {}
+
+  async sendEmail({ to, subject, message, link }: EmailOptions) {
```

Comment:
How to send message not by confirmation template?

Reply:
File changed to `template.hbs`. Provide `message` and `link` as context, so `.hbs` could paste this data into a template